### DISCUSSION TITLE
Can't unblock Roblox

### DISCUSSION THREAD

**Discussion by ch90045** [AdguardTeam/AdGuardHome#5733] [General]
Greetings,

i did set up AdGuard to do parental control for my kids - i like the feature that you can block individual services with the sliders. I do grant them daily some time with Roblox but that has stopped working for me.

They use iPads with all updates installed

Roblox is set to be allowed

![image](https://user-images.githubusercontent.com/2450761/232120664-00d46f98-e7e7-4cab-9d1e-0dabe0800ab6.png)

but i still get following

![image](https://user-images.githubusercontent.com/2450761/232121743-820662a3-98c5-45a8-bec2-f3e648b15b32.png)

And also i do see that 

![image](https://user-images.githubusercontent.com/2450761/232121927-491c3186-c726-4264-9e6d-3e713a15ab81.png)

So addlist i have are superseding the "Service allowance" - do i miss something essential here.

Any help would be much appreciated as the kids are really annoying not being able to play ROBLOX.

Christian

**Comment by ainar-g** [comment]
Hello.  We cannot reproduce this.  Are you blocking the services using client-specific settings or globally?  Are there any other allowed services?  (And, just to be clear, have you pressed the “Save” button at the bottom of the service list?  It's my personal pet peeve, heh.)

Regarding the metrics query, it's most likely not essential to Roblox's functioning.  Most likely a domain for tracking and statistics.

**Comment by ch90045** [comment]
I am doing this client specific and yes, a i am always hitting safe as
without it, when the whole thing was still working kids were haunting me
when i was NOT hitting safe ;)

Just to assure you, i am working in it and i know when things come to
effect or not.

Something which "bothers" me is the fact that typical adguard blocklists
are having roblox in there as well, so are blocklists overruling allowed
services lately?

Christai
----




Am Mo., 17. Apr. 2023 um 12:59 Uhr schrieb Ainar Garipov <
***@***.***>:

**Reply by ainar-g** [reply]
>  Something which "bothers" me is the fact that typical adguard blocklists are having roblox in there as well, so are blocklists overruling allowed services lately?

An unchecked service isn't really “allowed”, but rather “not blocked by the service blocker”.  This behavior has not changed in any recent releases, as far as I can see.  If you can reproduce this blocking of `www.roblox.com` reliably, you could [enable verbose logging][1], make a DNS query, and send the logs (possibly with your configuration file) to <devteam@adguard.com> with the words “AdGuard Home Issue 5733” in the subject line for us to take a look.

As for the second domain, `metrics.roblox.com`, *AdGuard DNS filter* is a generic ads/trackers filtering rule list, so it being there isn't really surprising.  You can [override][2] that rule from there with something like `@@|metrics.roblox.com`.  But again, I don't really think that the domain could be an issue.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog
[2]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#adblock-style

**Comment by ch90045** [comment]
totally agree on the metrics point ... i just was showing what was blocked
... i'll do the debug soon



Am Mo., 17. Apr. 2023 um 15:44 Uhr schrieb Ainar Garipov <
***@***.***>:
